packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -0,0 +1,348 @@
+    const [isElectionComplete, setIsElectionComplete] = useState(false);
+    const {
+        data: currentElection,
+        isLoading: isLoadingElection,
+        isError: isErrorElection,
+    } = useCurrentElection();
+
+    useEffect(() => {
+        if (currentElection?.electionState === ElectionStatus.Registration) {
+            setIsElectionComplete(true);
+        }
+    }, [currentElection]);
+
+    const isLoading = isLoadingElection;
+    const isError = isErrorElection;
+
+    if (isLoading || !currentElection) return <LoaderSection />;
+    if (isError) return <ErrorLoadingElection />;
+
+    const renderBanner = true; // TODO: get end time for video submission
+
+    return (
+        <FluidLayout
+            title="Election-video upload service"
+            banner={
+                renderBanner && (
+                    <EncryptionPasswordAlert
+                        promptSetupEncryptionKey={
+                            currentElection?.electionState !==
+                            ElectionStatus.Registration
+                        }
+                    />
+                )
+            }
+        >
+            {isElectionComplete ? (
+                <div>Election is complete.</div>
+            ) : (
+                <div className="divide-y">
+                    <Container>
+                        <Heading size={1}>
+                            Election-video upload service
+                        </Heading>
+                    </Container>
+                    <Container className="space-y-6 pb-5">
+                        <Text>
+                            Election video files are typically large and require
+                            many minutes to upload completely. Do not close this
+                            tab during the upload. A confirmation will be
+                            displayed when your upload is complete, along with a
+                            preview of your video.
+                        </Text>
+                        <Text>
+                            Be patient, and remember you have 48 hours from the
+                            beginning of the election to complete your election
+                            video uploads.
+                        </Text>
+                    </Container>
+                    <RoundVideoUploadList election={currentElection} />
+                </div>
+            )}
+        </FluidLayout>
+    );
+};
+
+export default RoundVideoUploadPage;
+
+const RoundVideoUploadList = ({ election }: { election: CurrentElection }) => {
+    const [ualAccount] = useUALAccount();
+    const {
+        data: communityGlobals,
+        isLoading: isLoadingGlobals,
+        isError: isErrorGlobals,
+    } = useCommunityGlobals();
+    const {
+        data: currentElection,
+        isLoading: isLoadingElection,
+        isError: isErrorElection,
+    } = useCurrentElection();
+    const {
+        data: ongoingElectionData,
```

Comment:
question: will this still be available after the election is over (_i.e._, no longer _ongoing_?) Does everything still work in that 48-hour post-election timeframe?